t/25-cache-service.t

```diff
@@ -109,6 +110,18 @@ sub test_sync {
 
     ok -e $expected, "expected file exists, run $run";
     is $expected->slurp, $data, "synced data identical, run $run";
+
+    my $rsync_request2 = $cache_client->rsync_request(from => "empty", to => "nowhewre");
+    ok !$cache_client->enqueue($rsync_request2);
+
+    wait_for_or_bail_out { $cache_client->status($rsync_request2)->is_processed } 'rsync';
+
+    my $status2 = $cache_client->status($rsync_request2);
+    is $status2->result, 'exit code 11', "exit code ok, run $run";
+    ok $status2->output, "output ok, run $run";
```

Comment:
One more thing you could do here is to test the actual output against a regex instead of just testing that it is not empty.

Reply:
isnt this in next line ?

Reply:
oh I'm stupid. Ok, but maybe actually add the `Try 1:` stuff to the regex to make sure it actually retries